includes/core/classes/class-setup.php

```diff
@@ -400,11 +400,14 @@ protected function create_tables(): void {
 	public function check_users_can_register(): void {
 		if (
 			filter_var( get_option( 'users_can_register' ), FILTER_VALIDATE_BOOLEAN ) ||
-			filter_var( get_option( 'gatherpress_suppress_membership_notification' ), FILTER_VALIDATE_BOOLEAN )
+			filter_var( get_option( 'gatherpress_suppress_membership_notification' ), FILTER_VALIDATE_BOOLEAN ) ||
+			filter_var( ! current_user_can( 'manage_options' ), FILTER_VALIDATE_BOOLEAN ) ||
+			false === strpos( get_current_screen()->id, 'gatherpress' )
+
 		) {
 			return;
 		}
-
+		wp_enqueue_style( 'gatherpress-admin-style' );
```

Comment:
I think this is fine for now, but feel like we are going to revisit this method as well as probably what is in the styling.

Reply:
Agree. 

While we have the beginning of support from core, there is still now way to persist a _Dismiss_ .

[![grafik](https://github.com/GatherPress/gatherpress/assets/198883/88e00438-3ee9-4ddf-be32-0697c5b80ef1)](https://make.wordpress.org/core/2023/10/16/introducing-admin-notice-functions-in-wordpress-6-4/)

https://make.wordpress.org/core/2023/10/16/introducing-admin-notice-functions-in-wordpress-6-4/